icepyx/core/variables.py

```diff
@@ -111,11 +108,7 @@ def __init__(
 
     @property
     def path(self):
-        if self._path:
-            path = self._path
-        else:
-            path = None
-        return path
+        return self._path or None
```

Comment:
Is this another case where there are "hidden" assumptions? Why doesn't this simplify to `return self._path if self._path else None`? To me, `return self._path or None` is like saying "yes" to "do you want an apple or an orange?" It's not obvious which one I should give you (and I can't give you both).

Reply:
There are no assumptions as Truth Value Testing with `or` and `and` is defined quite clearly at the beginning of the Python Standard Library docs before `int`, `list`, `dict`, etc. are defined.

Reply:
We can simplify this statement in two passes to make it a bit clearer:

```python
if self._path:
    path = self._path
else:
    path = None
return path
```

->

```python
if self._path:
    return self._path
else:
    return None
```

->

```python
return self._path or None
```

The `or` here is doing the lifting of the `else` condition -- if `self._path` is "Falsy", we move along to the next thing in the `or` operation.

Reply:
```suggestion
        return self._path if self._path else None
```